src/components/cc-tile-metrics/cc-tile-metrics.stories.js

```diff
@@ -0,0 +1,324 @@
+import './cc-tile-metrics.js';
+import './cc-tile-metrics.smart.js';
+import { makeStory, storyWait } from '../../stories/lib/make-story.js';
+import { enhanceStoriesNames } from '../../stories/lib/story-names.js';
+
+export default {
+  title: '🛠 Overview/<cc-tile-metrics>',
+  component: 'cc-tile-metrics',
+};
+
+const conf = {
+  component: 'cc-tile-metrics',
+  // language=CSS
+  css: `cc-tile-metrics {
+    margin-bottom: 1rem;
+  }`,
+};
+
+const baseItems = [
+  {
+    grafanaLink: 'https://grafana.example.com/small',
+    metricsLink: 'https://metrics.example.com/small',
+    style: 'max-width: 23.75em',
+  },
+  {
+    grafanaLink: 'https://grafana.example.com/medium',
+    metricsLink: 'https://metrics.example.com/medium',
+    style: 'max-width: 33.75em',
+  },
+  { grafanaLink: 'https://grafana.example.com/large', metricsLink: 'https://metrics.example.com/large' },
+];
+
+const ONE_HOUR = 60 * 60 * 1000;
+
+function fakeMetricData (numberOfPoints, usedValue, linearIncrease = false) {
+  return Array
+    .from({ length: numberOfPoints })
+    .map((_, index) => {
+      const randomShift = (usedValue !== 0) ? (Math.random() * 4) - 2 : 0;
+      const increase = (linearIncrease) ? index * 1.5 : 0;
+      console.log('value', usedValue + randomShift + increase);
+      return {
+        value: Math.min(usedValue + randomShift + increase, 100),
+      };
+    });
+}
+
+function now () {
+  return new Date().getTime();
+}
+
+function addTimestamp (array) {
+  const startTs = now();
+  return array.map((item, index) => {
+    return {
+      ...item,
+      timestamp: startTs + index * ONE_HOUR,
+    };
+  });
+}
+
+export const defaultStory = makeStory(conf, {
+  items:
+    baseItems.map((item) => ({
+      ...item,
+      metrics: {
+        state: 'loaded',
+        value: {
+          cpuData: addTimestamp(fakeMetricData(24, 25)),
+          memData: addTimestamp(fakeMetricData(24, 16)),
+        },
+      },
+    })),
+});
+
+export const highValues = makeStory(conf, {
```

Comment:
rename into dataLoadedWithHighValues so that the story name is well decorated in Storybook menu. (and apply to all story names)